manager/src/main/java/com/redhat/service/bridge/manager/models/DependencyStatus.java

```diff
@@ -0,0 +1,47 @@
+package com.redhat.service.bridge.manager.models;
+
+public class DependencyStatus {
+
+    boolean ready;
```

Comment:
I'll sleep on it... moving it to `Work` makes sense but we only get re-tries per `Work`.. which, I guess, from a User's perspective they only care that, for example, provisioning of a `Processor` was attempted X times and not that the `Processor` was attempted once but the `Connector` X times. 

IDK whether `ManagedResource.dependenciesReady` should be of type `ManagedResourceStatus` i.e. `ACCEPTED`, `READY`, `DEPROVISION`, `DELETED` etc. This would eliminate the need for `dependenciesReady` and `dependenciesDeleted` (that you do not mention).

Reply:
Moved to `Work` as proposed.